engine.py

```diff
@@ -89,61 +102,31 @@ def init_engine(self):
 
-        # import pyside QT UI libraries
-        self._init_pyside()
+        MOTIONBUILDER_2018_VERSION = 18000.0
+
+        if FBSystem().Version < MOTIONBUILDER_2018_VERSION:
+            # older versions are missing image plugins, hotpatch them in.
+            self._add_qt470_image_format_plugins_to_library_path()
 
         # motionbuilder doesn't have good exception handling, so install our own trap
         sys.excepthook = tank_mobu_exception_trap
 
-    def _init_pyside(self):
-        """
-        Handles the pyside init
-        """
-
-        pyside_folder = self.__get_pyside_folder()
-
-        # first see if pyside is already present - in that case skip!
-        try:
-            from PySide import QtGui
-        except:
-            # fine, we don't expect pyside to be present just yet
-            self.log_debug("PySide not detected - it will be added to the setup now...")
-        else:
-            # pyside was found. we will ensure the image format plugins are
-            # available
-            self.log_debug("PySide detected - the existing version will be used.")
-            self._add_image_format_plugins_to_library_path(pyside_folder)
-            return
-
-        if sys.platform == "win32":
-            pyside_path = os.path.join(self.disk_location, "resources", pyside_folder, "python")
-            sys.path.append(pyside_path)
-            dll_path = os.path.join(self.disk_location, "resources", pyside_folder, "lib")
-            path = os.environ.get("PATH", "")
-            path += ";%s" % dll_path
-            os.environ["PATH"] = path
-
-        else:
-            self.log_error("Unknown platform - cannot initialize PySide!")
-
-        # now try to import it
-        try:
-            from PySide import QtCore
-        except Exception, e:
-            self.log_error("PySide could not be imported! Apps using pyside will not "
-                           "operate correctly! Error reported: %s" % e)
-        else:
-            self._add_image_format_plugins_to_library_path(pyside_folder)
-
-    def _add_image_format_plugins_to_library_path(self, pyside_folder):
+    def _add_qt470_image_format_plugins_to_library_path(self):
         """
-        Add image format plugins to path so they can be loaded and used
+        Explicitly add image format plugins for qt 4.70/vs2010 compile
 
-        :param: pyside_folder Filesystem location where the plugins live
+        This is to patch an issue on older versions of Mobu where
+        these plugins are missing from the version of PySide that is
+        shipped with the DCC.
         """
-        from PySide import QtCore
+        from tank.platform.qt import QtCore
 
-        plugin_path = os.path.join(self.disk_location, "resources", pyside_folder, "qt_plugins")
+        plugin_path = os.path.join(
+            self.disk_location, 
+            "resources", 
+            "qt470_win64_vs2010", 
+            "qt_plugins"
+        )
 
         self.log_debug(
```

Comment:
[minor] Were you planning to update all of the legacy log_* calls? If so, missed this one! If not, ignore me!

Reply:
good point! i didn't plan on doing that. but i should :)